index.html

```diff
@@ -33,9 +33,9 @@ <h1 class="logo">My Logo</h1>
     </header>
     <main>
         <section id="portfolio">
-            <div class="shapes">
+            <div class="shapes flex">
                 <div class="portfolio-header">
-                    <h2>I’m Bahati<br>
+                    <h2 class="flex">I’m Bahati<br>
                        Glad to see </br>
                        you!
                     </h2>
```

Comment:
- [ ] [Optional] I think it would be better to make your webpage fully responsive, Currently on the screen size of 590 to 375 the webpage is not maximizing the width of the screen size 😨 please fix this for the best user experience 👍

| Your design| 
| --- |
|     ![Recording 2023-04-22 at 10 46 33](https://user-images.githubusercontent.com/83360636/233773731-188e92b4-b7bc-4c59-a3e6-085f4b52f534.gif)  |